cast/README.md

```diff
@@ -8,7 +8,7 @@ Home Assistant Cast is made up of two separate applications:
 ## Development
 
 - Run `script/develop_cast` to launch the Cast receiver dev server. Keep this running.
-- Navigate to http://localhost:8080 to start the launcher
+- Navigate to http://IP-OF-DEV-MACHINE:8080 to start the launcher
```

Comment:
This seems incorrect? The dev container would run local (or at least forward it to local).